src/command_modules/azure-cli-network/azure/cli/command_modules/network/commands.py

```diff
@@ -503,7 +503,7 @@ def _make_singular(value):
 # DNS RecordSetsOperations
 dns_record_set_path = 'azure.mgmt.dns.operations.record_sets_operations#RecordSetsOperations.'
 cli_command(__name__, 'network dns record-set list', custom_path + 'list_dns_record_set', cf_dns_mgmt_record_sets, transform=transform_dns_record_set_output)
-for record in ['a', 'aaaa', 'mx', 'ns', 'ptr', 'srv', 'txt']:
+for record in ['a', 'aaaa', 'mx', 'ns', 'ptr', 'srv', 'txt', 'caa']:
```

Comment:
The swagger spec change was made to `2017-09-01`, meaning it is not available from older one.  If true, then we need version check condition

Reply:
Yes, but DNS isn't versioned and is not part of any Stack profile. Thus until it is, there's only "latest"